packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -26,21 +25,18 @@ import DomainCategories from '../domain-categories';
 import {
 	PAID_DOMAINS_TO_SHOW,
 	PAID_DOMAINS_TO_SHOW_EXPANDED,
-	DOMAIN_SUGGESTIONS_STORE,
 	domainIsAvailableStatus,
 } from '../constants';
 import { DomainNameExplanationImage } from '../domain-name-explanation/';
 import { ITEM_TYPE_RADIO } from './suggestion-item';
 import type { SUGGESTION_ITEM_TYPE } from './suggestion-item';
+import { getDomainSuggestionsVendor } from '../utils';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-function
-const noop = () => {};
-
```

Comment:
As we're trying to removing `lodash` from this codebase (see p4TIVU-9Bf-p2) , I wonder what's the best approach here:

1. Use `lodash` functions, and wait for them to be replaced in a later refactor (that will likely add `noop` and `times` to the ``@automattic/js-utils` package
2. Keep the `noop` local implementation

Option 1 is my favourite, but just cc'ing @tyxla to confirm this is the right path

Reply:
The best way forward is to null-check the arguments where the `noop` is used. For example, if we do a plain:

```
const { onClick = noop } = this.props;
...

onClick();
```

We'd ideally check for its existence before calling it, as in:

```
if ( onClick ) {
	onClick();
}

That way we wouldn't need the `noop` default at all. Ideally, we won't be needing any `noop` as default arguments at all. 

Sometimes this isn't an option, though, as we're working with 3rd party code or for any other reason - in that case, declaring a local `noop` is the best option we have right now. 

I'd suggest avoiding `lodash` at all costs, especially for such trivial reasons, where using it really provides no value at all.

Reply:
Ah the `noop` was a rebase mistake were the version prior to the removal of lodash's `noop` was used instead. Eitherway, I'm getting rid of `const noop = () => {};` in favour of `if ( onClick ) { onClick(); }`. Thanks guys!